cms/djangoapps/contentstore/views/item.py

```diff
@@ -805,8 +812,18 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
         "format": xblock.format,
         "course_graders": json.dumps([grader.get('type') for grader in graders]),
         "has_changes": has_changes,
-        "override_type": override_type,
+        "actions": xblock_actions,
+        "explanatory_message": explanatory_message
     }
+
+    # Entrance exam subsection should be hidden. in_entrance_exam is inherited metadata, all children will have it.
```

Comment:
@asadiqbal08 -- I've left this one for you to work through -- please let me know if you need a hand.